### PR TITLE
Fix issue where reason of authentication error not delivered in OAuth…

### PR DESCRIPTION
**Pull request by tommy5oo**
…  (#128)
Fix issue where reason of authentication error not delivered in OAuth
Adjusting observer setting in LoadingActivity

### CODE REVIEW

File: nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/auth/NuguOAuthClient.kt [hunk 1]
```diff
@@ -143,7 +146,7 @@ internal class NuguOAuthClient(
      * Authorization flow
      * @param refreshAccessToken true is skip the LinkDevice
      * */
-    fun handleAuthorizationFlow(authCode: String?, refreshToken: String?): Boolean {
+    fun handleAuthorizationFlow(authCode: String?, refreshToken: String?) {
```
Comment (shshshl09) [inline nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/auth/NuguOAuthClient.kt]: add comment @throw

File: nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/auth/NuguOAuthClient.kt [hunk 2]
```diff
@@ -127,6 +127,9 @@ internal class NuguOAuthClient(
      * @return [AuthFlowState.STOPPING]
      */
     private fun handleStopping(): AuthFlowState {
+        if(credential.accessToken.isBlank()) {
+            throw ExceptionInInitializerError("accessToken is empty")
```
Comment (shshshl09) [inline nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/auth/NuguOAuthClient.kt]: ExceptionInInitializerError?

File: nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/net/HttpClient.kt [hunk 3]
```diff
@@ -88,7 +88,7 @@ class HttpClient(private val baseUrl: String) {
         } catch (e : SSLException) {
             return Response(HTTP_CLIENT_ERROR, "SSLException")
```
Comment (shshshl09) [inline nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/net/HttpClient.kt]: just ssl exception

File: nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/net/HttpClient.kt [hunk 4]
```diff
@@ -88,7 +88,7 @@ class HttpClient(private val baseUrl: String) {
         } catch (e : SSLException) {
```
Comment (shshshl09) [inline nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/net/HttpClient.kt]: check response strcuture

File: nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/auth/NuguOAuthClient.kt (hunk 1 above)
Comment (tommy5oo) [reply nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/auth/NuguOAuthClient.kt]: Add comment in handleRequestToken

File: nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/auth/NuguOAuthClient.kt (hunk 2 above)
Comment (tommy5oo) [reply nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/auth/NuguOAuthClient.kt]: Change ExceptionInInitializerError to Throwable

File: nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/net/HttpClient.kt (hunk 4 above)
Comment (tommy5oo) [reply nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/net/HttpClient.kt]: done

File: nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/net/HttpClient.kt (hunk 3 above)
Comment (tommy5oo) [reply nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/net/HttpClient.kt]: done

**Pull request merged**